src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -317,9 +322,10 @@ module TacoDependencyInstaller {
             });
         }
 
-        private promptUserBeforeInstall(): Q.Promise<any> {
+        private promptUserBeforeInstall(): Q.Promise<boolean> {
             this.buildInstallConfigFile();
 
+            var self = this;
```

Comment:
Our earlier guideline was to use the "var self = this". A quick search in the code base shows that we still use the "self" notation in at least 70 other places.

If we decide we want to completely remove any "self" notation from our code, I would be inclined to add that to tslint and do the necessary changes in a separate changeset... What do you think?